src/main/java/com/shopify/ShopifySdk.java

```diff
@@ -645,6 +597,75 @@ public ShopifyCustomer updateCustomer(final ShopifyCustomerUpdateRequest shopify
 		return shopifyCustomerRootResponse.getCustomer();
 	}
 
+	public ShopifyCustomer getCustomerById(final String id) {
+		final Response response = get(getWebTarget().path(CUSTOMERS).path(id));
+		final ShopifyCustomerRoot shopifyCustomerRootResponse = response.readEntity(ShopifyCustomerRoot.class);
+		return shopifyCustomerRootResponse.getCustomer();
+	}
+
+	public List<ShopifyCustomer> getCustomers(Response response) {
+		ShopifyCustomersRoot shopifyCustomersRootResponse = response.readEntity(ShopifyCustomersRoot.class);
+		return shopifyCustomersRootResponse.getCustomers();
+	}
+
+	public List<ShopifyCustomer> getCustomers(final int page) {
+		return getCustomers(page, DEFAULT_REQUEST_LIMIT);
+	}
+
+	public List<ShopifyCustomer> getCustomers(final int page, final int pageSize) {
+		final Response response = get(getWebTarget().path(CUSTOMERS)
+				.queryParam(LIMIT_QUERY_PARAMETER, pageSize)
+				.queryParam(PAGE_QUERY_PARAMETER, page));
+		return getCustomers(response);
+	}
+
+	public List<ShopifyCustomer> getCustomers(List<String> ids) {
+		final Response response = get(getWebTarget().path(CUSTOMERS)
+				.queryParam(IDS_QUERY_PARAMETER, StringUtils.join(ids, ",")));
+		return getCustomers(response);
+	}
+
+	public List<ShopifyCustomer> getCustomers(String sinceId) {
+		final Response response = get(getWebTarget().path(CUSTOMERS)
+				.queryParam(SINCE_ID_QUERY_PARAMETER, sinceId));
+		return getCustomers(response);
+	}
+
+	public List<ShopifyCustomer> getCustomers(final DateTime mininumCreationDate, final int page, final int pageSize) {
+		final Response response = get(getWebTarget().path(CUSTOMERS)
+				.queryParam(LIMIT_QUERY_PARAMETER, pageSize)
+				.queryParam(CREATED_AT_MIN_QUERY_PARAMETER, mininumCreationDate.toString())
+				.queryParam(PAGE_QUERY_PARAMETER, page));
+		return getCustomers(response);
+	}
+
+	public List<ShopifyCustomer> getCustomers(final DateTime mininumCreationDate, final DateTime maximumCreationDate,
+										final int page) {
+		return getCustomers(mininumCreationDate, maximumCreationDate, page, DEFAULT_REQUEST_LIMIT);
+	}
+
+	public List<ShopifyCustomer> getCustomers(final DateTime mininumCreationDate, final DateTime maximumCreationDate,
+										final int page, final int pageSize) {
+		final Response response = get(getWebTarget().path(CUSTOMERS)
+				.queryParam(LIMIT_QUERY_PARAMETER, pageSize)
+				.queryParam(CREATED_AT_MIN_QUERY_PARAMETER, mininumCreationDate.toString())
+				.queryParam(CREATED_AT_MAX_QUERY_PARAMETER, maximumCreationDate.toString())
+				.queryParam(PAGE_QUERY_PARAMETER, page));
+		return getCustomers(response);
+	}
+
+	public List<ShopifyCustomer> customerSearch(Response response) {
```

Comment:
Same for this method as the other one. Mark this as private and move towards bottom of file with other private methods.

Also, can we rename customerSearch to searchCustomer. A standard we try to follow is to start method names with verbs.